test/sanity/issue6099-AdSense-trouble/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "issue6099-AdSense-trouble",
+  "main": "https://xn--mrvk-xpa.tk/test/"
```

Comment:
The case should not depends on the public remote site, which could be unavailable in the future. Please hosts those files with a local http server in the case. See other case for example issue4877-inject-webview.

Reply:
Following case issue4877-inject-webview updated this case.